src/ui/display.h

```diff
@@ -194,15 +189,18 @@ bool ui_warn_high_fee(dispatcher_context_t *context);
 bool ui_validate_transaction(dispatcher_context_t *context,
                              const char *coin_name,
                              uint64_t fee,
+                             tx_ux_warning_t warnings,
                              bool is_self_transfer);
 
-bool ui_validate_transaction_simplified(dispatcher_context_t *context,
-                                        const char *coin_name,
-                                        const char *wallet_policy_name,  // can be NULL
-                                        uint64_t amount,
-                                        const char *address_or_description,
-                                        tx_ux_warning_t warnings,
-                                        uint64_t fee);
+void ui_validate_transaction_simplified_init(const char *wallet_policy_name,
+                                             unsigned int outputs_num,
+                                             tx_ux_warning_t warnings);
+void ui_validate_transaction_simplified_add(const char *coin_name,
+                                            uint64_t amount,
+                                            const char *address_or_description);
+bool ui_validate_transaction_simplified_start(const char *coin_name,
+                                              dispatcher_context_t *context,
+                                              uint64_t fee);
```

Comment:
While this whole header is generally not documented, these three functions need to be called in a specific order, so it might be worth documenting the correct usage (e.g.: first init, then add once or multiple times, then call _start).